apps/web/components/StatsCardGrid.tsx

```diff
@@ -0,0 +1,89 @@
+"use client";
+
+import type { LucideIcon } from "lucide-react";
+import { InfoIcon } from "lucide-react";
+import { Card } from "@/components/ui/card";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipTrigger,
+} from "@/components/ui/tooltip";
+
+const variants = {
+  blue: {
+    iconBg: "bg-blue-100 dark:bg-blue-900/50",
+    iconColor: "text-blue-600 dark:text-blue-400",
+  },
+  green: {
+    iconBg: "bg-green-100 dark:bg-green-900/50",
+    iconColor: "text-green-600 dark:text-green-400",
+  },
+  orange: {
+    iconBg: "bg-orange-100 dark:bg-orange-900/50",
+    iconColor: "text-orange-600 dark:text-orange-400",
+  },
+  purple: {
+    iconBg: "bg-purple-100 dark:bg-purple-900/50",
+    iconColor: "text-purple-600 dark:text-purple-400",
+  },
+  red: {
+    iconBg: "bg-red-100 dark:bg-red-900/50",
+    iconColor: "text-red-600 dark:text-red-400",
+  },
+  yellow: {
+    iconBg: "bg-yellow-100 dark:bg-yellow-900/50",
+    iconColor: "text-yellow-600 dark:text-yellow-400",
+  },
+} as const;
+
+export type StatVariant = keyof typeof variants;
+
+export type StatItem = {
+  icon: LucideIcon;
+  value: string | number;
+  title: string;
+  tooltip?: string;
+  variant?: StatVariant;
+  iconBg?: string;
+  iconColor?: string;
+};
+
+export function StatsCardGrid({ items }: { items: StatItem[] }) {
```

Comment:
So this component is just way over engineered. You use it in one place and it would be fine to just inline it. Not everything needs its own component file.

If you use it once, just inline where you're using it.